go.mod

```diff
@@ -15,18 +15,24 @@ require (
 	github.com/lib/pq v1.10.4
 	github.com/pelletier/go-toml v1.9.4
 	github.com/spiffe/go-spiffe/v2 v2.0.0
+	github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
```

Comment:
I thought metrics was already a part of our packages?  Or is this because we are now requiring it versus it getting pulled in indirect by other package?

Reply:
Yes, indirectly. Now it is being used directly